modules/apps/captcha/captcha-api/src/main/java/com/liferay/captcha/simplecaptcha/SimpleCaptchaImpl.java

```diff
@@ -89,6 +94,20 @@ public String getTaglibPath() {
 
 	@Override
 	public boolean isEnabled(HttpServletRequest httpServletRequest) {
```

Comment:
We should always returns true for these sensitive omniadmin portlet to enable captcha.
FYI, https://liferay.atlassian.net/browse/LPD-1558?focusedCommentId=2553257